front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -755,19 +777,25 @@ export async function submitAssistantBuilderForm({
             }
             return {
               type: "dust_app_run_configuration",
+              name: a.name,
```

Comment:
This will impact all actions here. What will be the value for non multi-actions flows?

Reply:
The default value (which come from the action default state).